### PR TITLE
Solution_exercise 1

### PR DESCRIPTION
**Pull request by JotGabiga**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 300px;
+    border-radius: 5%;
+    }
+
+#star
+{
+    font-size: 150px;
+    padding: 75px;
```
Comment (gabidzon) [inline style.css]: Whole div is clickable. It would be great if only star has this behaviour.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+}
+
+#star:hover
+{
+    transform: scale(1.03);
```
Comment (gabidzon) [inline style.css]: Can we change it to 1.05?

File: style.css [hunk 3]
```diff
@@ -0,0 +1,22 @@
+#container
+{
+    background-color:firebrick;
+    width: 300px;
+    height: 300px;
+    margin-left: auto;
```
Comment (gabidzon) [inline style.css]: You can set all margins in one line.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+	<meta charset="utf-8" />
+	<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+	
+	<title></title>
+	
+	<meta name="description" content="Opis w Google" />
+	<meta name="keywords" content="słowa, kluczowe, wypisane, po, porzecinku" />
+	<link rel="stylesheet" href="style.css" type="text/css"/> 	
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+	<script src="Script.js"></script>
```
Comment (gabidzon) [inline index.html]: Name of the file should start with lowercase.

File: Script.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+function changeClass(){
+    if (document.getElementById("star").classList.contains("glyphicon-star")){
```
Comment (gabidzon) [inline Script.js]: Can we extract `document.getElementById("star")` to variable? It is redundant to call document.getElementById method three times.

File: style.css (hunk 1 above)
Comment (JotGabiga) [reply style.css]: Done

**Pull request merged**